Dockerfile

```diff
@@ -1,37 +1,26 @@
 FROM ubuntu:18.04
```

Comment:
Is there a reason why you don't build upon the official adoptopenjdk docker image?

Reply:
It was easier to install the other dependencies on ubuntu that on the official adoptopenjdk image.

Reply:
ok!